plugins/security/nexus-ldap-plugin/src/main/java/org/sonatype/security/realms/ldap/internal/restlet/LdapTestAuthenticationPlexusResource.java

```diff
@@ -100,26 +94,19 @@ public Object put(Context context, Request request, Response response, Object pa
 
   @Override
   protected Object doPut(Context context, Request request, Response response, Object payload)
-      throws ResourceException,
-             InvalidConfigurationException
+      throws ResourceException
   {
 
     LdapAuthenticationTestRequest authRequest = (LdapAuthenticationTestRequest) payload;
 
-    CConnectionInfo connectionInfo = this.toLdapModel(authRequest.getData());
+    Connection connectionInfo = this.toLdapModel(authRequest.getData());
     String ldapServerId = null;
     if (request.getResourceRef() != null) {
       ldapServerId = request.getResourceRef().getQueryAsForm().getFirstValue("ldapServerId");
     }
     replaceFakePassword(connectionInfo, ldapServerId, ldapConfigurationManager);
 
-    ValidationResponse validationResponse = this.configurationValidator.validateConnectionInfo(
-        null,
-        connectionInfo);
-    // if the info is not valid throw
-    if (!validationResponse.isValid()) {
-      throw new InvalidConfigurationException(validationResponse);
-    }
+    new Validator().validate(connectionInfo);
```

Comment:
Why are we instantiating Validator here per request? It appears to still be @Named @Singleton and should be injected, not sure why that was removed?